applications/madini_tracechain.md

```diff
@@ -46,7 +46,48 @@ Bluegate now has specialization in products traceability based on accumulated kn
     
     If relevant, an indication of how you will integrate tools and features (e.g., Sync2/Connex, fee delegation protocols, MTT, etc) to enhance your project.
 
-Fee delegation protocol - Development in progress
```

Comment:
Already in place, any product udated into madinitracechain has fee delegation so local producrs dont have any additional costs publishing or using the traceability system